ac1/_sources/scatter_plots_and_correlation/toctree.rst

```diff
@@ -11,10 +11,10 @@ Scatter Plots and Correlation
    :caption: Contents
    :maxdepth: 2
 
-   introduction.rst
+   before_scatterplots.rst
```

Comment:
I think this PR needs to be combined with https://github.com/google/applied-computing-series/pull/19, because I don't see how the introduction and before_scatterplots fit together.

Reply:
Changes addressed in PR #19